static/app/components/superuserStaffAccessForm.tsx

```diff
@@ -39,8 +38,8 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
   constructor(props) {
     super(props);
     this.authUrl = this.props.hasStaff ? '/staff-auth/' : '/auth/';
+    this.authenticators = [];
```

Comment:
```suggestion
```
I'd recommend removing this and define it inline where you have the types , like this:
```
authenticators: Authenticator[] = [];
```
Which will do the same as what you have here.